code/game/mecha/working/ripley.dm

```diff
@@ -62,7 +62,7 @@
 			add_overlay(occupant ? "ripley-g-full" : "ripley-g-full-open")
 
 /obj/mecha/working/ripley/firefighter
-	desc = "Standart APLU chassis was refitted with additional thermal protection and cistern."
+	desc = "Standard APLU chassis was refitted with additional thermal protection and cistern."
```

Comment:
```suggestion
	desc = "Standard APLU chassis that was refitted with additional thermal protection and cistern."
```

This description makes no sense by itself

Reply:
May as well do the whole thing
```suggestion
	desc = "Standard APLU chassis that was refitted with additional thermal protection and a cistern."
```

Reply:
hmm yes this spelling mistake is made out of spelling mistake

Reply:
Do you think there should be an "A" at the start too?